src/classes/OPP_OpportunityContactRoles_TDTM.cls

```diff
@@ -325,33 +340,35 @@ public without sharing class OPP_OpportunityContactRoles_TDTM extends TDTM_Runna
             for (integer i=0; i<listOpps.size(); i++) {
                 Opportunity newOpp = listOpps[i];
                 Opportunity oldOpp = listOldOpps[i];
+
+                Boolean isIndividualAccount = (oppAccountDetails != null  && oppAccountDetails.containsKey(newOpp.AccountId)) ? oppAccountDetails.get(newOpp.AccountId).npe01__SYSTEMIsIndividual__c : null;
```

Comment:
seems like you addressed the code at line 234, but didn't change line 344.

Reply:
Good catch. I'll fix that one.